tools/ete/.shed.yml

```diff
@@ -0,0 +1,9 @@
+categories:
+- Phylogenetics
+description: Generates species tree from a list of species using the ETE Toolkit
+long_description: |
+	Generate a species tree from a list of species using the `ETE Toolkit`
```

Comment:
Please change tabs to spaces (also in following files). Tabs are forbidden in YAML.